resources/rpi/user-data

```diff
@@ -0,0 +1,74 @@
+#cloud-config
+# https://cloudinit.readthedocs.io/
+
+# Set the first part of the name to camera device.
+hostname: huntsmanpi000
+
+# If you have set up ssh on github you can pull down your
+# key automatically so that you can log into the unit without
+# a password. Change 'CHANGEME' to your username below.
+ssh_import_id:
+  - gh:huntsmanscope
+
+chpasswd:
+  expire: true
+  list:
+    - huntsman:huntsman
+
+ssh_pwauth: true
+
+groups:
+  - huntsman
+  - panoptes
+  - docker
+
+users:
+  - name: huntsman
+    gecos: Hunstman User
+    primary_group: huntsman
+    groups: users, admin, dialout, plugdev, docker, i2c, input, gpio, panoptes
+    sudo: ALL=(ALL) NOPASSWD:ALL
+    lock_passwd: false
+    shell: /bin/zsh
+
+## Update apt database and upgrade packages on first boot
+package_update: true
+package_upgrade: true
+
+byobu: enable
+
+ntp:
+  enabled: true
+  servers:
+    - time1.google.com
+    - time2.google.com
+    - time3.google.com
+    - time4.google.com
+
+## Install additional packages on first boot
+packages:
```

Comment:
Do we want to include the non-free/binary SoC drivers that are mentioned in the (apparently really, really out of date) Ubuntu Raspberry Pi wiki? I know we're running headless so maybe it doesn't matter.

Reply:
I think we'll probably fill this out a lot more. There's also the option below to run scripts, so we could just have the install script that changes.